EXPERIMENTAL: fix error: static_cast from 'NativeThreadId' (aka
'pthread *') to 'uint32_t' (aka 'unsigned int') is not allowed

Index: rts/System/MemPoolTypes.h
--- rts/System/MemPoolTypes.h.orig
+++ rts/System/MemPoolTypes.h
@@ -425,7 +425,7 @@ inline size_t StablePosAllocator<T>::Allocate(size_t n
 	if (positionToSize.empty()) {
 		size_t returnPos = data.size();
 		data.resize(data.size() + numElems);
-		myLog("StablePosAllocator<T>::Allocate(%u) = %u [thread_id = %u]", uint32_t(numElems), uint32_t(returnPos), static_cast<uint32_t>(Threading::GetCurrentThreadId()));
+		//myLog("StablePosAllocator<T>::Allocate(%u) = %u [thread_id = %u]", uint32_t(numElems), uint32_t(returnPos), static_cast<uint32_t>(Threading::GetCurrentThreadId()));
 		return returnPos;
 	}
 
